Public/Src/Utilities/Instrumentation/LogGen/Generators/AriaV2.cs

```diff
@@ -15,12 +16,18 @@ internal sealed class AriaV2 : GeneratorBase
         private const string GlobalInstrumentationNamespaceCommon = "global::BuildXL.Utilities.Instrumentation.Common";
         private const string GlobalTracing = "global::BuildXL.Tracing";
 
+        private const string EventCountMethodName = "EventCount";
+
         public override void GenerateLogMethodBody(LoggingSite site, Func<string> getMessageExpression)
         {
             m_codeGenerator.Ln("if ({0}.AriaV2StaticState.IsEnabled)", GlobalInstrumentationNamespaceCommon);
             using (m_codeGenerator.Br)
             {
-                m_codeGenerator.Lns("var eventData = new {0}.AriaEvent(\"{1}\", \"{2}\", \"{3}\")", GlobalInstrumentationNamespaceCommon, site.Method.Name, m_targetFramework, m_targetRuntime);
+                var tableName = OperatingSystemHelper.IsUnixOS && site.Method.Name == EventCountMethodName
+                    ? site.Method.Name + "MacOS"

```

Comment:
>MacOS [](start = 42, length = 5)

A comment on why we need to 'patch' the table name vs creating two events and #if for mac/windows vs joining the host os in telemetry would be nice.

Reply:
done